tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,35}-django111, py36-django111, quality, docs
+envlist = py{27,35,36}-django111, py35-django{20,21,22},quality, docs
```

Comment:
python36 with django20 ?

Reply:
I think our main requirement are python3.5 tests. Not sure if we need to check py3.6 too for later versions.

Reply:
I think we'll eventually want Python 3.6 + Django 2.2 tests, but honestly it depends on whether we upgrade to Ubuntu 18.04 (comes with Python 3.6) next or skip straight to 20.04 (will have either 3.7 or 3.8) after it comes out in April.